docs/user_workloads/creating_it_pref.md

```diff
@@ -0,0 +1,93 @@
+# Creating Instance Types and Preferences by using virtctl
+
+As of KubeVirt v1.0, you can use virtctl subcommands to create instance
+types and preferences.
+
+## Creating Instance Types
+
+The virtctl subcommand `create instancetype` allows easy creation of an instance
+type manifest from the command line. The command also provides several flags
+that can be used to create your desired manifest.
+
+There are two required flags that need to be specified:
+
+1. `--cpu`: the number of vCPUs to be requested
```

Comment:
```suggestion
1. `--cpu`: the number of vCPUs to be requested.
```

Reply:
Removed the dot in the line below. Actually, I like it more without dots.